spacer/extract_features_utils.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
Add tests for this file. Most can be moved from `test_caffe_utils.py`.

Reply:
added